blocks/feed/feed.js

```diff
@@ -139,7 +152,17 @@ export default async function decorate(block) {
     .take(blockCfg.count ? parseInt(blockCfg.count, 10) : 4)
     .toList();
   block.innerHTML = '';
-  const blockContents = resultParsers[blockType](results, blockCfg);
+  const meta = getMetadata('page-style');
```

Comment:
I'd suggest avoiding use of `page-style` for any purpose other than page level CSS styles.

Reply:
Yeah there seems to be a lot of CSS changes on this featured page and that's why put its CSS styles in featured.css & hence used page-style

Reply:
Sure working on it

Reply:
Fixed